<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QLabel" name="label_portname">
       <property name="text">
        <string>Port name : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_portname_3"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QLabel" name="label_baudrate">
       <property name="text">
        <string>Baut rate : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_bautrate_3">
       <property name="currentText">
        <string>115200</string>
       </property>
       <item>
        <property name="text">
         <string>1200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4800</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>57600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="label_databits">
       <property name="text">
        <string>Data bits : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_databits_3">
       <property name="currentText">
        <string>8</string>
       </property>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="label_parity">
       <property name="text">
        <string>Parity : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_parity_3">
       <item>
        <property name="text">
         <string>NoParity</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EvenParity</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>OddParity</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SpaceParity</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MarkParity</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_18">
     <item>
      <widget class="QLabel" name="label_stopbits">
       <property name="text">
        <string>Stop bits : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_stopbits_3">
       <property name="currentText">
        <string>TwoStop</string>
       </property>
       <item>
        <property name="text">
         <string>OneStop</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>OneAndHalfStop</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TwoStop</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_17">
     <item>
      <widget class="QLabel" name="label_flowcontrol">
       <property name="text">
        <string>Flow control : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_flowcontrol_3">
       <item>
        <property name="text">
         <string>NoFlowControl</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HardwareControl</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SoftwareControl</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
